examples/nextjs/package.json

```diff
@@ -8,9 +8,7 @@
     "lint": "next lint"
   },
   "dependencies": {
-    "next": "12.0.4",
-    "react": "^16.9.0",
-    "react-dom": "^16.9.0"
```

Comment:
Reusing root react/react-dom, cause there are 2 react instances and it's failing when we are using hooks as useState in the carousel. This won't happen when we are using the library from npm cause react/react-dom are in dev dependencies